Sniffs/UpdateURISniff.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace ET\ElegantThemes\Sniffs;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+class UpdateURISniff implements Sniff {
+
+	/**
+	 * Returns the token types that this sniff is interested in.
+	 * @return array(int)
+	 */
+	public function register() {
+		return [ T_COMMENT, T_DOC_COMMENT_STRING ];
+	}
+
+	/**
+	 * Processes the tokens that this sniff is interested in.
+	 *
+	 * @param File $phpcsFile                 The file where the token was found.
+	 * @param int  $stackPtr                  The position in the stack where
+	 *                                        the token was found.
+	 *
+	 * @return void
+	 */
+	public function process( File $phpcsFile, $stackPtr ) {
+        $tokens = $phpcsFile->getTokens();
+
+		if ( ! $this->isETUpdateURI( $tokens[ $stackPtr ]['content'] ) ) {
+			$phpcsFile->addError( "Invalid plugin update URI", $stackPtr, 'InvalidUpdateURI' );
```

Comment:
@Jronk @lanresmith Should we give better message with tips? Such as `"Invalid plugin update URI. Plugin submitted to Elegant Themes Marketplace should uses https://www.elegantthemes.com/ to avoid conflict with plugin sold on seller marketplace website." Or any better text, but we should tell them what is the correct one.

Reply:
OK. I used:

> Invalid plugin update URI. Plugins submitted to Elegant Themes Marketplace should use https://elegantthemes.com to avoid conflict with plugins sold on seller's website.